opensea.io/allowlist.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "../dapp-allowlist.schema.json",
+  "name": "Opensea",
+  "aliases": ["www.opensea.io", "app.opensea.io"],
```

Comment:
What do you have in mind for wildcards ? Do we handle theme here as another alias ? Or should they have their own field ?